templates/appveyor.yml.ejs

```diff
@@ -16,6 +16,13 @@ install:
 <%_ } else { _%>
   - npm install
 <%_ } _%>
+
+build: off
+
+build_script:
+  - ps: Write-Host “Prepack FLS plugin”
```

Comment:
FLS?

Reply:
Good catch, let me change it to be `Prepack plugin` or something generic.